client/app/scripts/superdesk/itemList/views/relatedItem-list-widget.html

```diff
@@ -31,12 +31,12 @@
 
     </div>
 
-    
+
    <!--  <div class="filter-box" ng-show="options.mode === 'detailed'">
         <span class="text" translate>View</span>
-        <button class="btn btn-light pull-left" 
+        <button class="btn btn-light pull-left"
```

Comment:
Maybe we can remove it, to keep code clean. This part of code was commented on April 7, 2015.